sensor/marker_radar_lidar_calibrator/CMakeLists.txt

```diff
@@ -3,6 +3,8 @@ cmake_minimum_required(VERSION 3.5)
 project(marker_radar_lidar_calibrator)
 
 find_package(autoware_cmake REQUIRED)
+find_package(OpenCV REQUIRED)
```

Comment:
Since we are using ament_auto, you should not need add either opencv nor ceres.
Please read how ament_auto works.
(opencv is actually added when you do `<depend>cv_bridge</depend>` it seems)

Reply:
Thanks! updated in the commit https://github.com/tier4/CalibrationTools/pull/161/commits/756eb81e0fbb31ec64c3b522c16018c0b59ef01f